tests/test/import/test.sh

```diff
@@ -34,6 +34,13 @@ rlJournalStart
         rlAssertGrep 'duration 5m' 'output'
     rlPhaseEnd
 
+    rlPhaseStartTest 'Check Makefile environment'
+        rlRun 'tmt test show | tee output'
+        rlAssertGrep 'AVC_ERROR: +no_avc_check' 'output'
+        rlAssertGrep '+no_avc_check' 'output'
```

Comment:
What's the point of this check? Isn't it just duplicating the preceding one?

Reply:
Thats a mistake, fixed.